bin/boilerplate/boilerplate.Dockerfile.txt

```diff
@@ -0,0 +1,14 @@
+# Copyright YEAR Istio Authors. All Rights Reserved.
```

Comment:
not an attorney - however - typically `All Rights Reserved` is incompatible with OSI approved licenses such as ASL2.

Reply:
Will remove that.

Reply:
@wackxu not to generate unnecessary red tape, however, any PR that wholesale changes all of the license headers IMO needs some form of legal review to ensure we aren't getting ourselves into trouble.   @jasmine-jaksic or @smawson can provide further guidance on the approach.

Cheers
-steve